www/install/insertBaseConf.sql

```diff
@@ -726,7 +730,6 @@ INSERT INTO `cb_type_field_relation` (`cb_type_id`, `cb_field_id`, `is_required`
 (11, 11, 1, 1),
 (11, 12, 1, 2),
 (11, 41, 0, 3),
-(13, 36, 0, 4),
```

Comment:
just a comment : this one is really hard to find

Reply:
move on line 850 for add js